oaknational/Oak-Web-Application
# Curric/lot picker UI update CUR-302

## Description

- Adds text-overflow to Box component
- Truncates span text to prevent overflowing
- Adds margin top to the examboard buttons

## Issue(s)

Fixes #CUR-302

## How to test

1. Go to https://deploy-preview-2034--oak-web-application.netlify.thenational.academy/teachers/curriculum
2. Change to mobile view and click on school phase picker 
3. You should see when exam board options wrap 16px spacing on the top
4. Input text truncates instead of spreading over 2 lines

## Screenshots

How it used to look (delete if n/a):
<img width="327" alt="Screenshot 2023-10-17 at 16 17 46" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/ce90c627-99e2-42c5-a3fe-c574e62282f5">
<img width="173" alt="Screenshot 2023-10-17 at 16 17 42" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/b57bfca6-8419-4e53-86a6-782f73e6fbef">


How it should now look:
<img width="213" alt="Screenshot 2023-10-17 at 16 16 23" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/5150863c-2b1e-4538-8f66-8365870a27ed">
<img width="330" alt="Screenshot 2023-10-17 at 16 16 20" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/da183310-aca2-484b-8d57-1446670da086">


## Checklist
